prompt/prompt-executor/prompt-executor-clients/prompt-executor-ollama-client/src/commonMain/kotlin/ai/koog/prompt/executor/ollama/client/dto/OllamaModels.kt

```diff
@@ -71,6 +71,9 @@ internal data class OllamaChatRequestDTO(
     @Serializable
     internal data class Options(
         val temperature: Double? = null,
+        val seed: Int? = null,
+        @SerialName("num_ctx") val numCtx: Int? = null,
```

Comment:
Doesn’t the Ollama json config assume the use of `snake_case` flag, so that we don’t have to use `@SerialName`?

Reply:
Ideally, all of this should be caught by tests, so I don’t see a big issue with it
Besides, we already use `namingStrategy` in other clients
So far, the only real advantage is a slight reduction in code

I don’t think it’s critical, since we can remove it or add it to the other clients at any time if needed